hw_diag/utilities/variant_definitions.py

```diff
@@ -0,0 +1,215 @@
+variant_definitions = {
```

Comment:
This is being copied from [another library](https://raw.githubusercontent.com/NebraLtd/helium-hardware-definitions/master/variant_definitions.py). Following Flask conventions, is this the right place? I've been putting this in a lib/` folder.

Reply:
Flask is mostly unopinionated about this kind of stuff, really this should be packaged and included via requirements.txt so it is easily re-usable in other projects, but thats out of scope for this story to convert the existing logic to Flask. I would suggest we create a story to find a better way on how to distribute and use this file.

Reply:
Agreed: https://github.com/NebraLtd/helium-hardware-definitions/issues/1